### ISSUE TITLE
Uncaught TypeError: Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by meeshypeeshy** [geoffthibeau/simple-drag-drop-text#3]
[Enter steps to reproduce below:]
1. select text
2. drag text
3. drop text.
4. uncaught error!

**Atom Version**: 0.201.0
**System**: Microsoft Windows 8.1 Enterprise
**Thrown From**: [simple-drag-drop-text](https://github.com/mark-hahn/simple-drag-drop-text) package, v0.2.2
### Stack Trace

Uncaught TypeError: Cannot read property 'start' of undefined

```
At C:\Users\marchester\.atom\packages\simple-drag-drop-text\lib\simple-drag-drop-text.coffee:59

TypeError: Cannot read property 'start' of undefined
  at SimpleDragDropText.drop (C:\Users\marchester\.atom\packages\simple-drag-drop-text\lib\simple-drag-drop-text.coffee:59:69)
  at SimpleDragDropText.clear (C:\Users\marchester\.atom\packages\simple-drag-drop-text\lib\simple-drag-drop-text.coffee:63:38)
  at HTMLBodyElement.<anonymous> (C:\Users\marchester\.atom\packages\simple-drag-drop-text\lib\simple-drag-drop-text.coffee:13:63)
  at HTMLBodyElement.jQuery.event.dispatch (C:\Users\marchester\AppData\Local\atom\app-0.201.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4435:9)
  at HTMLBodyElement.elemData.handle (C:\Users\marchester\AppData\Local\atom\app-0.201.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4121:28)

```
### Commands

```
     -0:04.0 core:undo (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "twilight-syntax"
    ],
    "destroyEmptyPanes": false,
    "projectHome": "d:\\devel\\active",
    "disabledPackages": [
      "language-make",
      "welcome"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
highlight-line, v0.10.2
minimap, v4.9.0
project-manager, v1.15.7
simple-drag-drop-text, v0.2.2
twilight-syntax, v0.7.0

# Dev
No dev packages
```

**Comment by Zireael07** [comment]
Was gonna report it when I saw this. Please fix ASAP, I can't work without it :)

**Comment by mark-hahn** [comment]
Thanks for the report.  I'll be getting to that package Sunday.

**Comment by mark-hahn** [comment]
Thanks again.  Fixed in 0.2.3.

**Comment by meeshypeeshy** [comment]
And thank you!
